pymeasure/adapters/__init__.py

```diff
@@ -36,5 +36,6 @@
 try:
     from pymeasure.adapters.serial import SerialAdapter
     from pymeasure.adapters.prologix import PrologixAdapter
+    from pymeasure.adapters.vxi11 import VXI11Adapter
```

Comment:
This should go in it's own try/catch clause to provide a relevant message. The PrologixAdapter is in there because it also requires PySerial.